app/javascript/components/host-initiator-form/host-initiator-form.schema.js

```diff
@@ -20,6 +20,13 @@ const loadStorages = (id) => API.get(`/api/providers/${id}?attributes=type,physi
     value: id,
   })));
 
+const loadGroups = (id) => API.get(`/api/physical_storages/${id}?attributes=host_initiator_groups`)
+  .then(({ host_initiator_groups }) => {
+    let groupOptions = host_initiator_groups.map(({ name }) => ({ label: name, value: name }));
```

Comment:
I think the value should be something unique that we could identify the record with (ideally `:id`) even if you only need the name [[here]](https://github.com/ManageIQ/manageiq-providers-autosde/pull/186/files#diff-41fa59bd46caf3a422d2250625c4a970d5a0eb9840ba37722b2a73b1407b171aR18) we might need more info in the future